zookeeper3/src/main/java/com/linecorp/armeria/server/zookeeper/ZooKeeperUpdatingListener.java

```diff
@@ -119,7 +120,9 @@ public static ZooKeeperUpdatingListenerBuilder builder(
     @Override
     public void serverStarted(Server server) throws Exception {
         final ZooKeeperRegistrationSpec registrationSpec = fillAndCreateNewRegistrationSpec(spec, server);
-        client.start();
+        if (client.getState() != CuratorFrameworkState.STARTED) {
```

Comment:
Oop... I didn't think of this case.

Reply:
Could you add this checking to the constructor of `ZooKeeperEndpointGroup`?

Reply:
I also don't know this case before using an external instance is set. T.T

Reply:
`ZooKeeperEndpointGroup`? Got it. will check it.